pylintrc

```diff
@@ -64,12 +64,13 @@
 # SERIOUSLY.
 #
 # ------------------------------
-# Generated by edx-lint version: 5.3.7
+# Generated by edx-lint version: 5.5.0
 # ------------------------------
 [MASTER]
 ignore = 
 persistent = yes
 load-plugins = edx_lint.pylint,pylint_django,pylint_celery
+django-settings-module = edxsearch.settings
```

Comment:
Can these changes to `pylintrc` be reverted?

Reply:
Ok cool, thanks @Ali-Salman29 , I didn't know how that file was maintained :)
Good to merge 👍